packages/core/content-releases/shared/contracts/release-actions.ts

```diff
@@ -38,3 +38,24 @@ export declare namespace CreateReleaseAction {
     error?: errors.ApplicationError | errors.ValidationError | errors.NotFoundError;
   }
 }
+
+/**
+ * GET /content-releases/:id/actions - Get all release actions
+ */
+export declare namespace GetReleaseActions {
+  export interface Request {
+    params: {
+      releaseId: Release['id']
+    };
+    query?: Partial<Pick<Pagination, 'page' | 'pageSize'>>;
+  }
+
+  export interface Response {
+    data: 
+      | ReleaseAction[]
+      | {
+        data: null;
+        error: errors.ApplicationError | errors.NotFoundError;
+      }
+  }
+}
```

Comment:
I think you might want to type it like above:

```
  export interface Response {
    data: ReleaseAction;
    error?: errors.ApplicationError | errors.ValidationError | errors.NotFoundError;
  }
```

Otherwise we found with Josh it doesn't play nicely with rtk-query